src/main/java/uk/gov/companieshouse/charges/data/service/ChargesService.java

```diff
@@ -235,6 +225,13 @@ public void deleteCharge(String contextId,
                                     HttpStatus.NOT_FOUND,
                                     "ChargeApi object doesn't exist for" + chargeId));
 
+            chargesRepository.deleteById(chargeId);
```

Comment:
We cannot change the order of deletes yet, if the call to chs-kafka-api fails this means there will be no data in Mongo for us to stream on the retry. We will be making delete changes in the future to allow for ```deleted``` stream events without ```deleted_data```  at which point we will be able to handle this order of calls.

Reply:
resolved